### ISSUE TITLE
Feature request: metadata editor and ability to organize books onto specific folders/pages.

### ISSUE THREAD

**Issue by Ninbusidu** [edrlab/thorium-reader#1896]
So I'm currently reading some Japanese light novels but sometimes the Metadata is broken and needs to be fixed and whenever a new volume comes out it gets really annoying trying to organize the books to be in order. 

I'm requesting a feature to drag the books up and down and to different pages or make folders for book series. A Metadata editor is not necessary but would be a great feature to have, you could edit the authors info, the cover, the name of the book, and the summary.

 Besides these two things I love the epub reader.

**Comment by gautierchomel** [comment]
Hello, would the [Tag feature](https://edrlab.github.io/thorium-reader-doc/220_organizing/223_tags/) do the job?

**Comment by gautierchomel** [comment]
Tag feature could do the job but adding metadata to a book could help making links from a book to another (version, collection, etc.) and more generally could be useful when annotating.

**Comment by dylan-k** [comment]
> A Metadata editor is not necessary but would be a great feature to have,

I would argue that a metadata editor is essential. If a book is imported with metadata that's missing/incorrect/illegible, it makes it pretty difficult to use the "All Publications" library.

**Comment by axelsimon** [comment]
I agree, some files will have bad or even no metadata and being able to fix that manually is a necessity.
While we can all agree that fixing this at the source, in catalogues, is the superior option, sometimes you just want to right click, edit title, and be done.

**Comment by axelsimon** [comment]
Shelves / sub-directories for the "Publications" sections is a very different feature and should be tracked in a different issue, @Ninbusidu.
